nomad-gcp/firewall.tf

```diff
@@ -47,7 +47,6 @@ resource "google_compute_firewall" "nomad-traffic" {
   }
 
   source_ranges = [data.google_compute_subnetwork.nomad.ip_cidr_range] #tfsec:ignore:google-compute-no-public-ingress
-  target_tags   = local.tags
```

Comment:
Removed the target binding with firewall as Nomad Client must connect to Nomad Server, either hosted in GCE or K8s Node.